tmt-app/src/constants/careerMajorList.js

```diff
@@ -177,15 +177,15 @@ export const MAJORS_LIST = [
     skills: ["People Management", "Conflict Resolution"],
     key: "education",
   },
-  {
+  '"humanities-and-cultures"': {
```

Comment:
Are the duplicate quotes intentional?

We can just use a single pair when using these as keys

Reply:
Yes, this is intentional. Using single pair works for most keys but there are a few that requires two pairs because of lint prettier. Making everything double would be a solution since we do not have to do additional logics to determine which keys have single vs two pairs.